autoware_launch/config/localization/pose_initializer.logging_simulator.param.yaml

```diff
@@ -0,0 +1,30 @@
+/**:
+  ros__parameters:
+    gnss_enabled: true
+    gnss_pose_timeout: 3.0 # [sec]
+    ndt_enabled: true
+    ekf_enabled: true
+    stop_check_enabled: false
+    stop_check_duration: 3.0 # [sec]
+
+    # from gnss
```

Comment:
Creating a covariance config in autoware_launch is redundant, so please separate parameter files.

Reply:
Thank you! Resolved: https://github.com/autowarefoundation/autoware_launch/pull/243/commits/c865d3ba4be625ed8a981727419944668da84a8a